packages/widget/src/constants.ts

```diff
@@ -1,5 +1,17 @@
+import { Environment } from '@buildwithsygma/sygma-sdk-core';
+
 export const DEFAULT_ETH_DECIMALS = 18;
 
 export const MAINNET_EXPLORER_URL = 'https://scan.buildwithsygma.com/transfer/';
 export const TESTNET_EXPLORER_URL =
   'https://scan.test.buildwithsygma.com/transfer/';
+
+export const SUBSTRATE_CHAIN_RPCS: { [env in Environment]: Array<string> } = {
```

Comment:
add comment (or event better adjust type) to only allow was connections

Reply:
you mean `ws` connections?

Reply:
```typescript
type WsUrl = `ws://${string}` | `wss://${string}`;
```
because later on, you are using this value as was connection